scripts/tests/run_tests

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
We don't need this file anymore, right?

Reply:
No we don't. When the tests passed I will clean up!